packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -68,6 +73,28 @@ export interface IDocumentSchema {
 	refSeq: number;
 
 	runtime: Record<string, DocumentSchemaValueType>;
+
+	// Other document related properties that are not related to the schema.
+	info: IDocumentSchemaInfo;
```

Comment:
I don't totally get the interplay between `IDocumentSchema` and `IDocumentSchemaCurrent` but I think this should be optional here.  I believe this is closer to the "incoming" variant, and the "Current" one is the "optimization" that says "I know I'll provide Info".

In fact, I would probably suggest flipping it to define `IDocumentSchemaCurrent` in terms of `IDocumentSchemaIncoming`, with `info` required.

Reply:
My observation that brought me here is that incoming ops use this `IDocumentSchema` type, but they could be from an old version without `info`.